### PR TITLE
includes zone model and collection

### PR DESCRIPTION
**Pull request by upadhyeammit**
This PR includes Zone model and collection.

### CODE REVIEW

**Comment by upadhyeammit** [comment]
@dengqinsi request to review !

**Comment by upadhyeammit** [comment]
@plribeiro3000 request you to help on reviewing and merging PR.

Thank You !

File: lib/fog/aliyun/models/compute/zones.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Fog
+  module Compute
+    class Aliyun
+      class Zones < Fog::Collection
+        model Fog::Compute::Aliyun::Zone
+
+        def all
+          data = Fog::JSON.decode(service.list_zones.body)['Zones']['Zone']
```
Comment (plribeiro3000) [inline lib/fog/aliyun/models/compute/zones.rb]: You should have a request to do the coding and decoding part of the process.
Could you try to use one instead?

Comment (upadhyeammit) [reply lib/fog/aliyun/models/compute/zones.rb]: If I see rest of the model implementation then decode method has been used. Do you mean there should be a common request to handle coding and decoding part for all models ? if not can you please elaborate more.

**Pull request closed without merging**